src/components/PositionDetails/LiquidationRangeInfo/style.ts

```diff
@@ -1,22 +1,23 @@
 import { makeStyles, Theme } from '@material-ui/core/styles'
-import { colors, typography } from '@static/theme'
+import { colors, newTypography, typography } from '@static/theme'
 
 const useStyles = makeStyles((theme: Theme) => ({
   infoTypeLabel: {
     textTransform: 'uppercase',
-    color: colors.invariant.lightInfoText,
-    ...typography.body1,
+    color: '#A9B6BF',
```

Comment:
use colors from theme

Reply:
comment about all uses of colors

Reply:
change done, new definition od color "textGrey" changed in the theme